components/service/home/ProgressBar.tsx

```diff
@@ -1,22 +1,25 @@
-import { Progress } from '@/components/ui/progress';
-
+// ProgressBar.tsx
 type Props = {
   value: number;
   title: string;
+  emoji?: string;
+  titleColor?: string;
 };
 
-export const ProgressBar = ({ value, title }: Props) => {
+export function ProgressBar({ value, title, emoji, titleColor }: Props) {
   return (
-    <div className="w-[90%] max-w-xs mx-auto space-y-2">
-      <div className="flex justify-between">
-        <p className="text-xs text-secondary font-semibold">{title}</p>
-        <p className="text-xs text-secondary font-semibold">{value}% 달성</p>
+    <div className="space-y-1">
+      <div className="flex items-center gap-2">
+        {emoji && <span className="text-lg">{emoji}</span>}
+        <span className={`font-semibold text-sm ${titleColor}`}>{title}</span>
+      </div>
+      <div className="w-full bg-gray-200 rounded-full h-2">
+        <div
+          className="bg-[#FBAA24] h-2 rounded-full transition-all duration-300"
+          style={{ width: `${value}%` }}
+        ></div>
```

Comment:
```suggestion
        />
```
div 안에 content 가 없다면 이렇게 바로 닫는태그로 닫아주시면 됩니당 :)

Reply:
수정 완료했습니다~!